doc/code.html

```diff
@@ -460,6 +463,43 @@ <h3 id="PackageNames">Package names</h3>
 Go's naming conventions.
 </p>
 
+<h2 id="Modules">Modules</h2>
```

Comment:
I don't think that modules themselves should really be covered in this document... this document is mainly just about getting some code running. It's especially awkward placement since it's placed in the middle of making our hello world package, and testing it out, and we just kinda have this weird tangent. Might be good for modules to just have their own document

Reply:
I agree it's in a poor position, I've moved it down to below remote packages but before the "What's Next" section. I still think we should touch upon it either briefly with a small section and a link out to the wiki, or even just a wiki link. 

I've also updated the remote packages section now that I'm home from work! Let me know if it looks good!